test/partiql-tests-runner/README.md

```diff
@@ -10,3 +10,11 @@ This package enables:
 Eventually, the `partiql-test-runner` module will replace the `partiql-pts` and `partiql-testscript` modules along with some other 
 tests in `lang` that were ported to `partiql-tests` 
 (see [partiql-lang-kotlin#789](https://github.com/partiql/partiql-lang-kotlin/issues/789)).
+
+## Run Conformance Tests Locally
```

Comment:
Could add that the default behavior will run tests in the `partiql-tests` submodule of `partiql-lang-kotlin`. And that the following commands can be used to run tests if `partiql-tests` is defined in a different directory

Reply:
ack